internal/modules/gravatar/gravatar.go

```
@@ -31,58 +34,97 @@ type GravatarUserAccounts struct {
 }
 
+func SearchGravatar(username string) {
+	theme.Yellow("[*] Searching Gravatar for ", username, "...").Println()
+
+	url := fmt.Sprintf("https://gravatar.com/%s.json", username)
+	resp, err := http.Get(url)
+	if err != nil {
+		theme.Redf("[-] Error fetching Gravatar data: %v", err).Println()
+		return
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)